LibVLCSharp.Uno/VideoView.Android.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Uno.UI;
+
+namespace LibVLCSharp.Platforms.UWP
+{
+    /// <summary>
+    /// Video view
+    /// </summary>
+    public partial class VideoView
```

Comment:
I try to avoid `partial class` as much as possible. Do you think we can come up with something clean without it?

Reply:
I updated the code in d481e7f32452bb8653f34b6723aaa4e523c2fca4. I use an abstract generic base class instead of a partial class.
But even if my class is no longer partial, I still have to let the `partial` keyword for Uno code generation.

Reply:
For more explanation on the Uno code generation, you can look at [this post](https://medium.com/@unoplatform/talkin-bout-my-generation-how-the-uno-platform-generates-code-part-1-under-the-hood-7664d83c4f90).